src/components/Landing.js

```diff
@@ -4,16 +4,18 @@ import '../assets/buttons.css';
 import './Landing.css';
 
 class Landing extends React.Component {
-  constructor ({ match }) {
-    super();
+  constructor (props) {
+    super(props);
     this.state = {
```

Comment:
You lifted the state up to the parent, is there a need for keeping an internal state for this component?

Reply:
The idea was to still keep the components as independent as they were, with their inner states, (just like the game component keeps his own state) In this case the state is equal in both `App` component and `Landing`, but do note that we still use it in the button's `disabled` property, and link destination in lines 97-99.
`Landing` can be changed into a stateless component, I personally don't find the overhead worth the change. What's your take on this?

Reply:
Why can't we use `disabled={!this.props.username}`?
I think it is worth it, as `Landing` has no need for internal state now that we lifted the state up.

Reply:
OK then, will do.